packages/storybook-addon-baseline-grid/package.json

```
@@ -0,0 +1,106 @@
+{
+  "name": "@canonical/storybook-addon-baseline-grid",
+  "version": "0.0.0",
+  "description": "Displays the baseline grid",
+  "keywords": [
+    "storybook-addons", "storybook-addons"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "-"
+  },
+  "type": "module",
+  "license": "MIT",
+  "author": "Bartek Szopka <bartek.szopka@canonical.com>",
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "import": "./dist/index.js",
+      "require": "./dist/index.cjs"
+    },
+    "./preview": {
+      "types": "./dist/index.d.ts",
+      "import": "./dist/preview.js",
+      "require": "./dist/preview.cjs"
+    },
+    "./preset": "./dist/preset.cjs",
+    "./manager": "./dist/manager.js",
+    "./package.json": "./package.json"
+  },
+  "files": [
+    "dist/**/*",
+    "README.md",
+    "*.js",
+    "*.d.ts"
+  ],
+  "scripts": {
+    "build": "tsup",
+    "build:watch": "npm run build -- --watch",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "run-p build:watch \"storybook --quiet\"",
+    "prerelease": "zx scripts/prepublish-checks.js",
+    "release": "npm run build && auto shipit",
+    "eject-ts": "zx scripts/eject-typescript.js",
+    "storybook": "storybook dev -p 6007",
+    "build-storybook": "storybook build"
+  },
+  "dependencies": {
+    "@storybook/icons": "^1.2.10"
+  },
+  "devDependencies": {
+    "@storybook/addon-essentials": "^8.4.7",
+    "@storybook/addon-interactions": "^8.4.7",
+    "@storybook/addon-links": "^8.4.7",
+    "@storybook/blocks": "^8.4.7",
+    "@storybook/react": "^8.4.7",
+    "@storybook/react-vite": "^8.4.7",
+    "@storybook/test": "^8.4.7",
+    "@types/node": "^22.10.1",
+    "@types/react": "^19.0.1",
+    "@types/react-dom": "^19.0.2",
+    "@vitejs/plugin-react": "^4.3.4",
+    "auto": "^11.1.1",
+    "boxen": "^7.1.1",
+    "dedent": "^1.5.1",
+    "npm-run-all2": "^6.2.3",
+    "prettier": "^3.3.3",
```

Should be replaced by Biome.

```suggestion
    "@biomejs/biome": "^1.9.4",
    "@canonical/biome-config": "^0.4.0-experimental.0",
```

---

Will do it in a follow up.